### PR TITLE
doc(gradientci): add v2 PS-12340

### PR DESCRIPTION
**Pull request by bbatha**

### CODE REVIEW

**Comment by dte** [comment]
I tried updating the paperspace/mnist-sample repo to use v2 syntax in a PR here: https://github.com/Paperspace/mnist-sample/pull/36/files#diff-d0a97cc474107660358ce41838ae0093

It looks like the commit went through but it is reading from the old config file (see the `G1` being used vs the `K80` here https://www.paperspace.com/console/projects/pr1igb9nc/experiments/ezvfcod86ctee )

Screen Shot 2019-12-30 at 8 15 32 PM

File: projects/gradientci-v2.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+* `only-pulls`: only perform this check on pull requests
+* `if-not-found`: return a default status if job has no data for `<identifier>`, defaults to “failure”
+* `comment-on-pr`: include this metric in a summary content if the metrics were generated from a pull-request
+
+## Examples
+
+### Repositories
+
+* [MNIST in TensorFlow](https://github.com/paperspace/mnist-sample)
+* [SDK Pipeline](https://github.com/paperspace/TBD)
```
[the remark this answers is not in the document]
Comment (dte) [reply projects/gradientci-v2.md]: this one? https://github.com/Paperspace/e2e-tests or https://github.com/Paperspace/e2e-monitoring ?

[the remark this answers is not in the document]
Comment (bbatha) [reply projects/gradientci-v2.md]: No its a different example, I'm not sure its in a repo yet. It should be a python script with many calls to the api.

**Comment by bbatha** [comment]
@dte the v2 supporting version was just deployed to dev this morning. The config files are hosted in the same location and the v1 syntax is still functional.

File: SUMMARY.md [hunk 2]
```diff
@@ -42,7 +42,8 @@
 * [Create a Project via the GUI](projects/create-a-project.md)
 * [Create a Project via the CLI](projects/create-a-project-using-cli.md)
 * [Deleting a Project](projects/deleting-a-project.md)
-* [GradientCI](projects/gradientci.md)
+* [GradientCI](projects/gradientci-v2.md)
+  * [GradientCI V1 (Deprecated)](projects/gradientci-v2.md)
```
Comment (joberly) [inline SUMMARY.md]: Is old V1 doc link still just `projects/gradientci.md`?

**Review by joberly — APPROVED** [review]

**Comment by dkobran** [comment]
@bbatha We need to start testing this feature asap to make sure it's working/stable. @curtis-ps will be helping with this but we need to have docs to follow to test this.  Can you help us tmrw morning to put together the docs/examples/test plan?

**Comment by bbatha** [comment]
@dkobran these docs are good to go and contain numerous simple examples. They are just waiting to merge as @mkutsovsky had some specific full repo examples that he wanted to make sure were included. Running those examples should be part of the test plan as well.

**Comment by dkobran** [comment]
@mkutsovsky Can you QA this in the staging team?

**Review by mikuts — APPROVED** [review]

**Pull request merged**